fixtures/coverall/tests/bindings/test_coverall.rb

```diff
@@ -52,6 +55,7 @@ def test_none_dict
   end
 
   def test_constructors
+    GC.start
```

Comment:
AFAICT the Ruby garbage collector is not deterministic in freeing objects when they go out of scope, so we have to manually trigger collections at key points where we want to test whether things have been cleaned up correctly.

Reply:
Unfortunately, this is accurate 👍